pkg/utils/utils.go

```diff
@@ -186,6 +187,14 @@ func GetShareDir() string {
 	return shareDir
 }
 
+func GetScriptsDir() string {
+	scriptsDir := os.Getenv("SCRIPTS_DIR")
```

Comment:
i don't think there's a need/want to let users override this value

Reply:
this is inline with our current pattern for workingdir... user's could only override if they go unmanaged, edit the deployment, kick the pod

Reply:
i still think its unnecessary to provide this, it was never configurable before and not sure what it actually provides...

Reply:
The issue here is that method under test `CreateOrUpdateCertificates` relied on a hardcoded string. However, we need to be able to setup a proper dir for unit and e2e tests independent of container fs layout.